src/modules/accounting/accounting.py

```diff
@@ -327,3 +312,103 @@ def get_extra_data(self, blockstamp: ReferenceBlockStamp) -> ExtraData:
             orl.max_items_per_extra_data_transaction,
             orl.max_node_operators_per_extra_data_item,
         )
+
+    @lru_cache(maxsize=1)
+    def _get_generic_extra_data(self, blockstamp: ReferenceBlockStamp) -> tuple[ValidatorsForOperator, ValidatorsForOperator, OracleReportLimits]:
```

Comment:
Let's make this a type
`type GenericExtraData`

Reply:
Done